homeassistant/components/velux/__init__.py

```diff
@@ -25,7 +39,23 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         LOGGER.exception("Can't connect to velux interface: %s", ex)
         return False
 
-    entry.runtime_data = pyvlx
+    device_identifier = (DOMAIN, f"gateway_{entry.entry_id}")
+    entry.runtime_data = VeluxData(pyvlx=pyvlx, gateway_device_id=device_identifier)
```

Comment:
So we'd be storing this in the runtime data, but: everywhere where we have access to the runtime data, we have access to the entry id, so we don't really need to store the device identifier tuple in the runtime data

Reply:
In a way having the test accessing the device with a "hard coded" identifier also helps test that we don't accidentally change it